its/ruling/src/test/resources/jdk6/squid-S2189.json

```diff
@@ -0,0 +1,8 @@
+{
+'jdk6:java/lang/ref/Finalizer.java':[
+157,
```

Comment:
I'm not sure about these findings, as they are in the body of a `run()` method, from a `Thread`. This behavior is somehow expected.